package com.luminoso.usersmanagement.exceptions

class UserAlreadyExistException(message: String? = null, cause: Throwable? = null) : RuntimeException(message, cause)